Source/Extensions/Blazorise.Video/Video.razor.cs

```diff
@@ -632,7 +632,7 @@ public Task NotifyQualityChange( int? quality )
     /// <summary>
     /// If the default controls are used, you can specify which settings to show in the menu.
     /// </summary>
-    [Parameter] public string[] SettingsList { get; set; } = new string[] { VideoSettingsType.Captions, VideoSettingsType.Quality, VideoSettingsType.Speed, VideoSettingsType.Loop };
+    [Parameter] public VideoSettingsType[] SettingsList { get; set; } = new VideoSettingsType[] { VideoSettingsType.Captions, VideoSettingsType.Quality, VideoSettingsType.Speed, VideoSettingsType.Loop };
```

Comment:
Actually I meant this to be flags that you can set like: 
`SettingsList = VideoSettingsType.Captions | VideoSettingsType.Quality | VideoSettingsType.Speed | VideoSettingsType.Loop`

Reply:
I mean, it's just a bit more elegant api and one less allocation. 
I don't care for it, if internally it's harder to handle. (Another option would be to have a custom internal property that would convert the values into valid strings the js expects, anyway, if it's more complex to do it, not worth it in relation to the benefits I believe.)

Reply:
I would argue that using flags is even harder. For example, how would you dynamically combine flags? With an array type, you can just `Add` or `Remove` values from a temporarily lost and set it like `SettingsList="someList.ToArray()"`.

Reply:
It's just the usage of bitwise operators:
Add
`|=`
Remove
`^=`
And there's an helper `HasFlag` so you don't have to use a bitwise operation to calc whether it exists.

I guess there's the chance that devs might not be quite as familiar with flags and how they work, yea let's keep the array, nvm.

Reply:
I am familiar with flags. But not all devs are. That's my concern.